app/src/main/java/com/xa/xpensauditor/misc_diff.java

```diff
@@ -0,0 +1,1374 @@
+//     String Uid;
+//     String day1, month1, year1;
+//     String day2, month2, year2;
+//     int d1, m1, y1;
+//     int d2, m2, y2;
+//     private Firebase RefTran3;
+//     @Override
+//     protected void onCreate(Bundle savedInstanceState) {
+//         super.onCreate(savedInstanceState);
+//         setContentView(R.layout.activity_month_wise_vis);
+//         auth = FirebaseAuth.getInstance();
+//         mRootRef=new Firebase("https://xpensauditor-g11-default-rtdb.firebaseio.com/");
+//         mRootRef.keepSynced(true);
+//         Uid=auth.getUid();
+//         RefUid= mRootRef.child(Uid);
+//         RefName = RefUid.child("Name");
+//         RefEmail=RefUid.child("Email");
+
+
+//         // RefUid.child("DateRange").child(String.valueOf(month + "-" + year)).child("Transactions").child(Tid).child("Amount").setValue(Amount);
+//         Bundle bundle = getIntent().getExtras();
+//         String month2= bundle.getString("m2");
+//         String year2= bundle.getString("y2");
+//         String day2= bundle.getString("d2");
+//         String month1= bundle.getString("m1");
+//         String year1= bundle.getString("y1");
+//         String day1= bundle.getString("d1");
+//         int m2=Integer.parseInt(month2);
+//         int y2=Integer.parseInt(year2);
+//         int d2=Integer.parseInt(day2);
+//         int m1=Integer.parseInt(month1);
+//         int y1=Integer.parseInt(year1);
+//         int d1=Integer.parseInt(day1);
+
+//         Cartesian cartesian = AnyChart.column();
+//         AnyChartView anyChartView = findViewById(R.id.any_chart_view);
+
+//         List<DataEntry> data = new ArrayList<>();
+//         {
+
+//             Boolean f = false;
+//             Calendar calendar = Calendar.getInstance();
+
+//             int thisYear = calendar.get(Calendar.YEAR);
+//             // Log.d(TAG, "# thisYear : " + thisYear);
+
+//             int thisMonth = calendar.get(Calendar.MONTH)+1;
+//             //Log.d(TAG, "@ thisMonth : " + thisMonth);
+
+//             int thisDay = calendar.get(Calendar.DAY_OF_MONTH);
+//             //Log.d(TAG, "$ thisDay : " + thisDay);
+//             if (thisYear > y2) {
+//                 f = true;
+//             } else {
+//                 if (thisYear == y2) {
+//                     if (thisMonth > m2) {
+//                         f = true;
+//                     } else {
+//                         if (thisMonth == m2) {
+//                             if (thisDay >= d2) {
+//                                 f = true;
+//                             }
+//                         }
+//                     }
+//                 }
+//             }
+
+//             if (f) {
+//                 System.out.println("Day_Wise   "+m2 +"    "+y2);
+//                 for( ii=m1;ii<=m2;ii++){
+//                 DatabaseReference reference= FirebaseDatabase.getInstance().getReference().child(Uid).child("DateRange").child(String.valueOf(ii + "-" + y2)).child("Transactions");
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 183).